Code/Engine/Core/World/Implementation/World.cpp

```diff
@@ -1070,7 +1070,7 @@ void ezWorld::UpdateSynchronous(const ezArrayPtr<ezInternal::WorldData::Register
       continue;
 
     {
-      EZ_PROFILE_SCOPE(updateFunction.m_sFunctionName);
+      EZ_PROFILE_SCOPE(updateFunction.m_sFunctionName.GetData());
```

Comment:
There should be no need to explicitly write the GetData() call here. If it doesn't compile, the tracy macros need to be adjusted.